src/mgmt/srv_pool.c

```diff
@@ -110,46 +109,34 @@ ds_mgmt_hdlr_pool_create(crt_rpc_t *rpc_req)
 	char				id[DAOS_UUID_STR_SIZE];
 	d_rank_list_t			*rank_list;
 	d_rank_list_t			tmp_rank_list = {0};
-	d_rank_t			ranks_array[TMP_RANKS_ARRAY_SIZE];
-	unsigned int			ranks_size;
+	d_rank_t			ranks_array[32];
```

Comment:
why remove the constant definition here? just because it's only used in this file?

Reply:
Not only is it used just in this file, it is local to a single function - to declare the size of the array. Subsequent occurrences can easily be replaced by ARRAY_SIZE (see line 126).